### PR TITLE
feat(ui5-multi-input, ui5-multi-combobox): implement popover keyboard handling

### PR DESCRIPTION
**Pull request by Okiana**
Implements BGSOFUIRILA-3652

### CODE REVIEW

File: packages/main/src/MultiComboBox.ts [hunk 1]
```diff
@@ -674,6 +674,7 @@ class MultiComboBox extends UI5Element {
 
 	_onkeydown(e: KeyboardEvent) {
 		const isArrowDownCtrl: boolean = isDownCtrl(e);
+		const isCtrl = !!(e.metaKey || e.ctrlKey);
```
Comment (ndeshev) [inline packages/main/src/MultiComboBox.ts:677]: event.metaKey and event.ctrlKey are boolean by default you don't need to cast them as such so just 
`const isCtrl = e.metaKey || e.ctrlKey;` should be enough

you could also at a type as the variable above it

File: packages/main/src/MultiInput.ts [hunk 2]
```diff
@@ -220,6 +220,8 @@ class MultiInput extends Input {
 
 		const target = e.target as HTMLInputElement;
 		const isHomeInBeginning = isHome(e) && target.selectionStart === 0;
+		const isCtrl = !!(e.metaKey || e.ctrlKey);
```
Comment (ndeshev) [inline packages/main/src/MultiInput.ts:223]: same as the comment for the ui5-multicombo-box

File: packages/main/test/specs/MultiInput.spec.js [hunk 3]
```diff
@@ -303,6 +303,32 @@ describe("MultiInput Truncated Token", () => {
...
+
+	it("should open popover on keyboard combination ctrl + i", async () => {
+		const mi = await $("#truncated-token");
+		const token = await mi.$("ui5-token");
+		const rpoClassName = await getTokenizerPopoverId("truncated-token");
+		const rpo = await browser.$(`.${rpoClassName}`).shadow$("ui5-responsive-popover");
+
+		await mi.click();
+		await mi.keys(["Control", "i"]);
+		assert.ok(await rpo.getProperty("opened"), "Focused MI - More Popover should be opened");
```
Comment (ndeshev) [inline packages/main/test/specs/MultiInput.spec.js:315]: more popover?  It is n-more in general, but when in this case with only one token it is just a token popover, no n-more button is shown as it is only one long truncated token.

But it should be good to add a test covering the case with multiple tokens.

**Comment by ndeshev** [comment]
You could add "related to" or  "implements" before the BL item ID in the initial comment for more human-friendliness :) 
Fix the` ui5-multi-input` tag in the title.

**Comment by ndeshev** [comment]
Don't forget to revert the WDIO configuration changes in the next commit :)

File: packages/main/test/specs/MultiInput.spec.js (hunk 3 above)
Comment (Okiana) [reply packages/main/test/specs/MultiInput.spec.js]: done

File: packages/main/src/MultiComboBox.ts [hunk 4]
```diff
@@ -674,6 +674,7 @@ class MultiComboBox extends UI5Element {
 
 	_onkeydown(e: KeyboardEvent) {
 		const isArrowDownCtrl: boolean = isDownCtrl(e);
+		const isCtrl: boolean = e.metaKey || e.ctrlKey;
```
Comment (ndeshev) [inline packages/main/src/MultiComboBox.ts:677]: We should check if is it ok this feature to work with both CTRL and CMD combinations on Mac

**Review by ndeshev — APPROVED** [review]

**Pull request merged**